docs/src/merge.md

```diff
@@ -31,13 +31,6 @@ accident_counts = accident_list.groupby("latimes_make_and_model").size().reset_i
 survey = pd.read_csv("https://raw.githubusercontent.com/palewire/first-python-notebook/main/docs/src/_static/faa-survey.csv")
 ```
 
-Before you do anything, take a peek at it with with the `head`.
```

Comment:
This is of course good practice, but a bit of a spoiler for the data cleaning we're introducing.